test/client/api.js

```diff
@@ -427,4 +426,24 @@ ClientApi.heartbeat = function (origin) {
   return (new ClientApi(origin)).doRequest('GET', origin + '/__heartbeat__')
 }
 
+function getQueryString (options) {
+  var qs = ''
```

Comment:
One more thing re: `querystring.stringify`, we'd probably want to pass in a new object containing the known parameters, just in case any stray properties are hanging around on the `options` object.